### PR TITLE
mysql: add 2 infoschema related errors for TiDB

### PR DESCRIPTION
**Pull request by bb7133**
### What problem does this PR solve? 
Some errors in TiDB do not have an error code. Add parts of them.

Side effects

 - Breaking backward compatibility

### CODE REVIEW

File: mysql/errname.go [hunk 1]
```diff
@@ -949,6 +949,8 @@ var MySQLErrName = map[uint16]string{
 	ErrInvalidTxn:                 "invalid transaction",
 	ErrEntryTooLarge:              "entry too large, the max entry size is %d, the size of data is %d",
 	ErrNotImplemented:             "not implemented",
+	ErrInfoSchemaExpired:          "Information schema is out of date: schema failed to update in 1 lease",
```
Comment (winkyao) [inline mysql/errname.go]: "Please make sure TiDB can connect to TiKV nodes"

Comment (lonng) [reply mysql/errname.go]: I think the error message starting with the lower case is more idiomatic.

Comment (bb7133) [reply mysql/errname.go]: Addressed

Comment (bb7133) [reply mysql/errname.go]: @lonng I agree with you, however, most of the existing error message has upper case prefix. So it's better to keep this style.

**Pull request merged**